src/main/java/io/cryostat/net/reports/SubprocessReportGenerator.java

```diff
@@ -137,12 +116,15 @@
                         // See https://github.com/cryostatio/cryostat/issues/287
```

Comment:
Why is #324 no longer necessary? Can this comment be removed then?

Reply:
#324 is about trying to fine-tune and eke out every last bit of optimized performance out of the subprocess generator, but that was always a compromised position to start out with. It's just a heavyweight operation by nature, and setting the subprocess heap size to take up as much space within the Cryostat container without causing Cryostat to OOM while also being able to process large reports requires an impossible balance between minimizing Cryostat's overall resource footprint and over-provisioning Cryostat enough that the subprocess generator can run and produce results.

Since this PR removes the resource footprint of report generation from the Cryostat container in the new preferred deployment mode, #324 can be dropped. And anyway, since the OOM score work that I did for the forked subprocess a while ago, the report generator should always be killed on out of memory and not the main Cryostat process. So the stability and robustness concern that was originally there in #324 has also been solved as well.